src/Command/AnalyseCommand.php

```diff
@@ -55,6 +55,7 @@ protected function configure(): void
 				new InputOption('autoload-file', 'a', InputOption::VALUE_REQUIRED, 'Project\'s additional autoload file path'),
 				new InputOption('error-format', null, InputOption::VALUE_REQUIRED, 'Format in which to print the result of the analysis', null),
 				new InputOption('generate-baseline', null, InputOption::VALUE_OPTIONAL, 'Path to a file where the baseline should be saved', false),
+				new InputOption('allow-empty-baseline', null, InputOption::VALUE_NONE, 'Do not error out when the generated baseline is empty'),
```

Comment:
This is missing a default

Reply:
You can't set a default on an input option which has no value.

Reply:
Oh right, haven't realized that 👍